### ISSUE TITLE
start_kernel() was not discussed earlier, though part 6 of Booting seems to imply it was

### ISSUE THREAD

**Issue by pdelong42** [0xAX/linux-insides#743]
In part 6 of the kernel booting process [1], you reference start_kernel() in init/main.c as if it had been discussed previously.  However, this is the first time I've seen either that function or file mentioned (since there doesn't appear to have been any mention of either of them in parts 1-5).  There are plenty of references to arch/x86/boot/main.c [3], but they're not directly related as far as I can tell.

I wasn't disoriented for too long, but it makes me wonder if I missed something, or if there was earlier material you omitted by mistake.

[1] https://github.com/0xAX/linux-insides/blob/master/Booting/linux-bootstrap-6.md
[2] https://github.com/torvalds/linux/blob/v4.16/init/main.c
[3] https://github.com/torvalds/linux/blob/v4.16/arch/x86/boot/main.c

**Comment by 0xAX** [comment]
@pdelong42 yes you're right. IIRC this part was written after `initialization` chapter, so the `start_kernel` already was in my mind :)

I'll try to find time to improve this.

**Comment by 0xAX** [comment]
This should be fixed in https://github.com/0xAX/linux-insides/pull/906
